modules/nf-core/gecco/run/meta.yml

```diff
@@ -56,7 +56,7 @@ output:
       pattern: "*.clusters.tsv"
   - gbk:
       type: file
-      Description: Per cluster GenBank file (if found) containing sequence with annotations. Will not be generated if no hits are found.
+      description: Per cluster GenBank file (if found) containing sequence with annotations. Will not be generated if no hits are found.
```

Comment:
```suggestion
      description: Per-cluster GenBank file (if found) containing sequence with annotations. Will not be generated if no hits are found.
```

Reply:
Module reviews are not required for pipeline review, so we'll update this nf-core module before next release ;)